apkid/rules/dex/packers.yara

```diff
@@ -70,3 +70,25 @@ rule medusah_appsolid_dex : packer
   condition:
     is_dex and $loader and $main_activity
 }
+
+rule apkguard : packer
```

Comment:
I love all the stuff you're submitting, but let's keep everything organized into 1 PR = 1 thing. Makes it easier to review and get stuff merged.